Add gulp to handle build process.

afaur opened the pull request:
This allows the project to support the `gulp` command to build the files in the dist folder.
The gulp script will also build a min.js version that's not gziped.

Commit:
Add gulp to handle build process.

bayleedev commented on package.json:
```diff
@@ -20,6 +20,15 @@
   "devDependencies": {
-    "browserify": "^11.2.0"
+    "browserify": "^11.2.0",
+    "gulp": "^3.9.0",
```
Since `gulp` is a dependency, why should the user install it globally? They could do a command:

```
./node_modules/bin/gulp
```

Or simply you could make a command:

```
npm run build
```

This is more universal since all node applications can have commands, but not all contain gulp.

afaur replied:
Good idea I will add this.

afaur commented:
Added a npm command `build` that will use the `gulp.js` included in the package `node_modules` folder.  Also updated the instructions for building in the readme file.  Now you can build using `npm run build`.

bayleedev commented:
Did you mean to commit `dist/jquery-meteor-blaze.min.js`?

murillo128 commented:
Right, remove the .min from the commit and I will merge it.

afaur commented:
Oops, sorry about that.  Removing now.

afaur commented:
Okay, got that out.  Good looking out, had tested the build process when I added the build script to the package.json and had accidentally committed it.

bayleedev commented:
:+1:

murillo128 commented:
Thnxs for the PR!

Also, if you have any other suggestion/comment/fix/improvement, please feel free to open a new issue.
